src/cljs/org/broadinstitute/firecloud_ui/page/workspaces.cljs

```diff
@@ -322,22 +344,23 @@
         (:error-message @state) [:div {:style {:color "red"}}
                                   "FireCloud service returned error: " (:error-message @state)]
         :else [comps/Spinner {:text "Loading workspaces..."}])])
+   :get-initial-state
```

Comment:
I generally add this before :render. I basically put these in the same order they are called in the lifecycle.